### PR TITLE
Development

### PR DESCRIPTION
**Pull request by tommygonzaleza**

### CODE REVIEW

File: breathecode/registry/management/commands/subscribe_asset_repos.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import logging
```
Comment (jefer94) [inline breathecode/registry/management/commands/subscribe_asset_repos.py:1]: Does this command work?

File: breathecode/registry/management/commands/subscribe_asset_repos.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+from django.core.management.base import BaseCommand
+from ...models import Asset
+from breathecode.authenticate.models import AcademyAuthSettings
+from breathecode.monitoring.actions import subscribe_repository
```
Comment (jefer94) [inline breathecode/registry/management/commands/subscribe_asset_repos.py:5]: This command must be added to the scheduler

File: breathecode/services/google_cloud/big_query.py [hunk 3]
```diff
@@ -471,9 +473,42 @@ def schema_difference(cls, old_schema: Schema, new_schema: Schema) -> BigQuerySe
...
+                if len(new_field._fields) > 0:
+                    res.append(new_field)
 
             elif new_field != old[key]:
                 res.append(new_field)
 
         return res
+
+    @classmethod
+    def merge_schema(cls, diff: Schema, schema: Schema) -> BigQuerySet:
```
Comment (jefer94) [inline breathecode/services/google_cloud/big_query.py:485]: This is the novelty in activity app

File: breathecode/registry/actions.py [hunk 4]
```diff
@@ -261,6 +259,19 @@ def set_blob_content(repo, path_name, content, file_name, branch='main'):
     return repo.update_file(file[0].path, f'Update {file_name}', content, file[0].sha)
 
 
+def generate_screenshot(url: str, dimension: str = '1200x630', **kwargs):
```
[the remark this answers is not in the document]
Comment (tommygonzaleza) [reply breathecode/registry/actions.py:262]: Yes, it's working we tested on the previous meeting

**Pull request merged**